README.md

```diff
@@ -12,19 +12,34 @@ git clone https://github.com/NREL/mappymatch.git
 
 Change directory into the mappymatch folder:
 ```
-cd path/to/mappymatch   # from here, your path to mappymatch is likely .\mappymatch\
+cd path/to/mappymatch
+# from here, your path to mappymatch is likely .\mappymatch\
 ```
 
 Then, use the environment.yml file (which was downloaded when you cloned the repo) to install dependencies:
+
+---> Question about this ordering (for @nreinicke or admin).
+Should this instalation be performed before or after navigating into the subfolder mappymatch/mappymatch?
+The environment.yml file lives in the outer directory under the primary MappyMatch folder.
+
+
+
+
 ```
 conda env create -f environment.yml
+
+# Is there a reason we are not using the line below? It allows us to specify the name and the filename in one line.
+# conda env create -n mappymatch --file environment.yml
 ```
 
 To activate the mappymatch environment:
 ```
 conda activate mappymatch
+
 ```
 
+Finally, be sure to change your Python interpreter path to the ./mappymatch/bin/python interpreter
```

Comment:
I found that my interpreter path was not defaulting to the correct conda environment. This may be a helpful clarification for users who are new to conda or struggling to get up and running.